### ISSUE TITLE
Accessibility: Color contrast insufficient – UI component

### ISSUE THREAD

**Issue by vikrampm1** [woothemes/woocommerce#52132]
**Prerequisites (mark completed items with an [x]):**
- [x] I have checked that my issue type is not listed here https://github.com/woocommerce/woocommerce/issues/new/choose
- [x] My issue is not a security issue, support request, bug report, enhancement or feature request (Please use the link above if it is).

### Issue Description:

The star rating icons fail color contrast against the background color. See color contrast analysis: https://webaim.org/resources/contrastchecker/?fcolor=DFA843&bcolor=FFFFFF 

The selected and unselected states also only have a 1.3:1 contrast ratios between them. It's possible that some people will not be able to distinguish between selected and unselected stars. https://webaim.org/resources/contrastchecker/?fcolor=C6C6C6&bcolor=E6A238 

This is a meaningful piece of information that does not currently have a text alternative visible on the page. Rating information may not be available to low vision users.

![Image](https://github.com/user-attachments/assets/f10b05ae-3388-4db5-a28c-c50f0cfd40cb)

![Image](https://github.com/user-attachments/assets/40254f31-4ff5-4c1f-a918-acb85a92a8f4)

### Component or Page: 

Component: All Reviews Block     

### Name affected element:

Star rating icon filled color

### Relevant code snippets:

`<div class="wc-block-review-list-item__rating wc-block-components-review-list-item__rating"><div class="wc-block-review-list-item__rating__stars wc-block-components-review-list-item__rating__stars" role="img" aria-label="Rated 5 out of 5"><span style="width: 100%;">Rated <strong class="rating">5</strong> out of 5</span></div></div>`

### Recommended fix:

Sufficient fix: Change the colors to have a minimum color contrast of at least 3:1, the minimum required color contrast for UI components, or add a border to the stars that provides the minimum contrast ratio. Ensure there is atleast 3:1 contrast between the selected and unselected stars.

Better: Increase color contrast to 4.5:1 or 7:1. Or, add a text alternative on visible on the page near the stars ""5 out of 5 stars"" or ""rating: 5/5"" that communicates the same information to low vision users that the stars communicate. Change unselected stars to have an outline only and not be solidly filled so they can more clearly differentiate between selected and unselected stars.
